students/swietczak_monika/lesson_06_dicts_tuples_sets_args_kwargs/args_sum.py

```diff
@@ -0,0 +1,9 @@
+def sumall(*args):
```

Comment:
Please use PEP8 convention for function names: lowercase with underscores
sum_all(*args)